fhir-persistence-schema/docs/DB2MultiTenancy.md

```diff
@@ -1,16 +1,225 @@
-# The IBM FHIR Server - Db2 Multi-Tenancy
+# The IBM FHIR Server - Multi-Tenancy on Db2
 
-This document outlines the schema design and implementation for the IBM FHIR Server's Db2 Multi-tenancy feature. 
+This document outlines the schema design and implementation for the IBM FHIR Server's Multi-tenancy feature on Db2.
 
 The Multi-tenancy feature has two main areas of focus: 
 - Tenant Provisioning  
 - Security
 
-## Multi-Tenancy Schema Design
+This document assumes familiarity with the general [IBM FHIR Server schema design](SchemaMigration.md).
+
+# Multi-Tenancy Concepts and Design
+
+The IBM FHIR Server design supports three levels of multi-tenancy:
+
+1. Separate database per tenant
+2. Separate data schema per tenant
+3. (Db2-only) multiple tenants per schema, partitioned with row-based access control and per-tenant tablespaces.
```

Comment:
I'll make this more clear. We do support 1 and 2 because each tenant gets its own fhir-server-config datasource set up. We can even mix database types.

Reply:
done